pom.xml

```
@@ -93,7 +93,46 @@
             <artifactId>jquery-ui</artifactId>
             <version>1.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.21</version>
```

It makes JMH bundled to the final plugin distribution. the scope should be test

```suggestion
            <version>1.21</version>
            <scope>test</scope>
```